docs/quickstart.md

```diff
@@ -146,6 +146,84 @@ Delete the table to clean up:
 bin/uc table delete --full_name unity.default.my_table
 ```
 
+## Manage models in Unity Catalog using MLflow
+
+Unity catalog supports the management and governance of ML models as securable assets.  Starting with 
+[MLflow 2.16.1](https://mlflow.org/releases/2.16.1), MLflow offers integrated support for using Unity Catalog as the 
+backing resource for the MLflow model registry.  What this means is that with the MLflow client, you will be able to 
+interact directly with your Unity Catalog service for the creation and access of registered models.
+
+## Setup MLflow for usage with Unity Catalog
+
+In your desired development environment, install MLflow 2.16.1 or higher:
+
+```sh
+$ pip install mlflow>=2.16.1
+```
+
+The installation of MLflow includes the MLflow CLI tool, so you can start a local MLflow server with UI by running the command below in your terminal:
+
+```sh
+$ mlflow ui
+```
+
+It will generate logs with the IP address, for example:
+
+```
+(mlflow) [master][~/Documents/mlflow_team/mlflow]$ mlflow ui
+[2023-10-25 19:39:12 -0700] [50239] [INFO] Starting gunicorn 20.1.0
+[2023-10-25 19:39:12 -0700] [50239] [INFO] Listening at: http://127.0.0.1:5000 (50239)
+```
+
+Next, from within a python script or shell, import MLflow and set the tracking URI and the registry URI.
+
+```python
+import mlflow
+
+mlflow.set_tracking_uri("http://localhost:5000")
+mlflow.set_registry_uri("uc:http://localhost:8080")
```

Comment:
The tracking uri does not seem to like localhost.  I will change both to be 127.0.0.1 as that works for both.